src/rdkafka_conf.c

```diff
@@ -847,7 +848,11 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "Callback to verify the broker certificate chain.",
           _UNSUPPORTED_SSL
         },
-
+        { _RK_GLOBAL, "ssl.certificate.fetch_cb", _RK_C_PTR,
```

Comment:
Use all-underscore property names for properties that require programmatical setters, as to differentiate them from string properties.

Reply:
Oh, I see we missed this for verify_cb, damn. Okay, let's keep fetch_cb as is then.